examples/next/pages/ui/components/liveness/components/useLiveness.ts

```diff
@@ -39,13 +48,19 @@ export function useLiveness() {
   const stopLiveness = () => {
     setLivenessActive(false);
     setGetLivenessResponse(null);
-    // mutate();
+    mutate();
   };
 
   const handleGetLivenessDetection = async (sessionId) => {
-    // const response = await API.get('BYOB', `/liveness/${sessionId}`, {});
-    // setGetLivenessResponse(response);
-    // return { isLive: response.isLive };
+    const response = await get({
+      apiName: 'BYOB',
+      path: `/liveness/${sessionId}`,
+      options: {},
+    }).response;
+    const { body } = response;
+    const livenessResponse = await body.json();
```

Comment:
any chance this can throw?

Reply:
I believe it can but this is also just in the sample app